LandLord-api/src/main/java/biz/princeps/landlord/api/ILandLord.java

```diff
@@ -29,6 +30,14 @@
      */
     JavaPlugin getPlugin();
 
+    /**
```

Comment:
This can be removed by merging  #50 which makes ILandLord a plugin and makes all delegates obsolete

Reply:
Nice!